data/incipias/incipias.txt

```diff
@@ -0,0 +1,372 @@
+
+phrase "hex digit"
+	word
+		"0"
+		"1"
+		"2"
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+		"a"
+		"b"
+		"c"
+		"d"
+		"e"
+		"f"
+
+fleet "Hicemus Standard Surveilance"
+	government "Hicemus"
+	names "hicemus military"
+	cargo 0
+	personality
+		heroic vindictive
+	variant 10
+		"Nimbus"
+	variant 4
+		"Nimbus" 2
+	variant 4
+		"Venta"
+
+fleet "Incipias Standard Mining"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 0
+	personality
+		timid mining harvests coward
+	variant 2
+		"Nimbus (Mining)"
+	variant 5
+		"Nimbo Stratus"
+	variant 5
+		"Nimbo Stratus" 2
+	variant 1
+		"Nimbo Stratus"
+		"Nimbus (Mining)"
+
+fleet "Incipias Standard Trade"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 50
+	personality
+		confusion 40
+		timid appeasing
+	variant 5
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbus (Civil)"
+	variant 4
+		"Nimbo Cirrus" 2
+	variant 3
+		"Gero"
+	variant 2
+		"Gero" 2
+	variant 2
+		"Gero"
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbo Cirrus" 2
+		"Nimbus (Civil)"
+
+outfitter "Hicemus"
+	"Plasma Discharger"
+	"Stagnation Beam"
+	"Gas-Class Shield"
+	"Liquid-Class Shield"
+	"Metallic Hydrogen Cell"
+	"Cloud Piercer"
+
+	"MH Blue-Class Engine"
+	"MH Blue-Class Steering"
+	"MH Green-Class Engine"
+	"MH Green-Class Steering"
+	"MH Red-Class Engine"
+	"MH Red-Class Steering"
+
+shipyard "Hicemus"
+	"Nimbo Cirrus"
+	"Nimbo Stratus"
+	"Nimbus"
+	"Gero"
+
+news "Incipias common"
+	location
+		government "Conlatio" "Hicemus"
+	name
+		word
+			"Incipias Captain"
+			"Incipias Crew Member"
+	message
+		phrase
+			"incipias common hard defined"
+			"incipias common praise"
+
+phrase "incipias common"
+	phrase
+		"incipias common hard defined" 1
+		"incipias common praise" 5
+		"incipias common interest" 3
+		"incipias common advise" 3
+
+phrase "incipias common hard defined"
+	word
+		"You say you left this home-system? One day, I will also see other home-systems."
+		"The before-present was a more interesting time to fly a spaceship. Nowadays, a lot of people have one."
+		"I just got my crewmember-ship-license renewed. The government controls them strictly."
+		"There is nothing more exciting than to switch to turbine mode and fly through the landscape-clouds of our origin-home."
+		"It has been some time since the last great knowledge-breakthrough has been achieved in technological research-understanding. The government promises us that they are working on something, but my guess is that they have no clue what to do."
+		"I saw a plasma discharger with a leak-crack not so long ago. The ship looked good from the outside, but the inside was a firestorm."
+		"I have never seen something like your species. Why don't you float like us?"
+		"I was on a mining mission-job not so long ago when I saw a government ship drop a small box. The box accelerated very fast in one direction and disappeared with a flash."
+
+phrase "incipias common praise"
+	word
+		"These technologies"
+		"These ships"
```

Comment:
These ships of mine?

Reply:
Where is the problem?

Reply:
There isn’t one, my first read-through got it wrong. You can resolve this.